acceptance/framework/flags/flags.go

```diff
@@ -115,6 +116,13 @@ func (t *TestFlags) init() {
 	flag.BoolVar(&t.flagEnableCNI, "enable-cni", false,
 		"If true, the test suite will run tests with consul-cni plugin enabled. "+
 			"In general, this will only run against tests that are mesh related (connect, mesh-gateway, peering, etc")
+	flag.BoolVar(&t.flagRestrictedPSAEnforcementEnabled, "restricted-psa-enforcement-enabled", false,
```

Comment:
Nit: This flag should be flipped the other way to match the patterns to maybe `-enable-restricted-psa-enforcement`?

I think setting PSA as an 'infra' thing is fine as we have to do the same with CNI; there are specific setups for Kind and the clouds.

Reply:
Oh, do you mean change the flag name and also have the tests create and configure the consul namespace with PSA enforcement? (rather than requiring the namespace be configured with PSA enforcement before the test is run)

Reply:
I changed this flag name (but kept the behavior of needing to set the restricted label on the namespace).